README.md

```diff
@@ -209,7 +209,9 @@ binci --cleanup-all
 
 ## Environment Variables (`env <array>`)
 
-Setting `env` array items will expose environment variables in the primary instance or services. These entries can be raw strings or use `${VAR}` notation, where `VAR` is an environment variable on the host machine to use. Entries should use the format `<ENV_VAR>=<VALUE>`
+Setting `env` array items will expose environment variables in the primary instance or services. These entries can be raw strings or use `${VAR}` notation, where `VAR` is an environment variable on the host machine to use. Entries should use the format `<ENV_VAR>=<VALUE>`.

```

Comment:
Looks good 👍 , Could you add an implementation example in the `env` block on L87?

Reply:
Done!

Reply:
![you rock](https://media.giphy.com/media/UsK5GqD3e0huU/giphy.gif)